lib/livebook/notebook/cell/elixir.ex

```diff
@@ -24,8 +24,8 @@ defmodule Livebook.Notebook.Cell.Elixir do
   """
   @type output ::
           :ignored
-          # Regular text, adjacent such outputs can be treated as a whole
-          | binary()
+          # IO output, adjacent such outputs are treated as a whole
+          | {:stdout, binary()}
```

Comment:
Should we support :__pruned__ here? Or is it only internal?

```suggestion
          | {:stdout, binary() | :__pruned__}
```